app/views/user_journey/confirmation.html.erb

```diff
@@ -3,6 +3,10 @@
 <%= render 'shared/certificate_details', certificate: @certificate %>
 
 <h1 class="govuk-heading-l"><%= t 'user_journey.adding_certificate_to_config' %></h1>
+  
+<div class="govuk-inset-text">
```

Comment:
Why not `<p>`?

Reply:
Its whats in the design system guide as the example 🤷🏽‍♂️